src/main/webapp/destination.css

```diff
@@ -1,22 +1,106 @@
-.required {
-  color: red;
-}
-
 .required::after {
   content: "*";
+  color: red;
 }
 
 .place {
   cursor: pointer;
-  padding-top: 5px;
-  padding-bottom: 5px;
+  margin-bottom: 5px;
   border-style: solid;
   border-width: 2px;
-  border-color: grey;
+  border-color: white;
   width: 200px;
 }
 
-#map {
+.content {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  text-align: center;
+}
+
+.form-box {
+  background-color: #5d5179;
+  flex-grow: 1;
+  overflow: auto;
+  border: 2px solid white;
+  border-radius: 12px;
+  margin: 20px auto;
+  color: white;
+}
+
+#search-results {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.map-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#map-area {
   height: 300px;
   width: 800px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#desc-input {
+  width: 400px;
+  margin-top: 5px;
+}
+
+#name-input {
+  margin-top: 5px;
+}
+
+#hint1-input {
+  margin-top: 5px;
+  width: 299px;
```

Comment:
Should this also be 300px? And could you apply all of the common styling for the hint inputs either by adding a `.hint-input` class, or making a single rule that applies to them all, e.g.
```
#hint1-input,
#hint2-input,
#hint3-input {
  // blah blah
}